web.jsf12/external/binaries-list

```diff
@@ -1 +1,18 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
 B9DB1A789C301F1D31DD6CC524DA2EBD7F89190D jsf-1.2.zip
```

Comment:
Maybe, download jsf-api.jar from [Maven Central](https://mvnrepository.com/artifact/javax.faces/jsf-api/1.2) ? It seems that the module uses it only.
https://github.com/apache/incubator-netbeans/blob/26cd220e505b04448d77a054ed6d7d29cf420120/web.jsf12/nbproject/project.properties#L18-L26

#### binaries-list
```
224B0C2CCA97240305DE96B3108D2E7397FBFC4D javax.faces:jsf-api:1.2
```
#### project.properties
```
release.external/jsf-api-1.2.jar=modules/ext/jsf-1_2/jsf-api.jar
```
@matthiasblaesing Is this correct?

Reply:
Hm - its a pitty - nearly every build of the jsf-1.2 API is present on maven central, but not the version (1.2_04-b20-p03) i the referenced jar. However from my experience the APIs are pretty stable, so I would tage the next bugfix revision: https://search.maven.org/artifact/javax.faces/jsf-api/1.2_05/jar.

That would mean, that the license file needs to be updated. For the project.properties I would remove lines 18-21 and lines 23-26. They look superflous and wrong (the API is not AL licensed).

Reply:
@matthiasblaesing Thanks!

@vikasprabhakar Could you please fix them? Please see the Matthias' comment.

Reply:
@matthiasblaesing  The following jars of web.jsf12 module are used in web.jsf12ri module
release.../web.jsf12/external/jsf-1.2.zip!/jsf-1.2/commons-beanutils.jar=modules/ext/jsf-1_2/commons-beanutils.jar
release.../web.jsf12/external/jsf-1.2.zip!/jsf-1.2/commons-collections.jar=modules/ext/jsf-1_2/commons-collections.jar
release.../web.jsf12/external/jsf-1.2.zip!/jsf-1.2/commons-digester.jar=modules/ext/jsf-1_2/commons-digester.jar
release.../web.jsf12/external/jsf-1.2.zip!/jsf-1.2/commons-logging.jar=modules/ext/jsf-1_2/commons-logging.jar
release.../web.jsf12/external/jsf-1.2.zip!/jsf-1.2/jsf-impl.jar=modules/ext/jsf-1_2/jsf-impl.jar

Could you provide the license details of following jar files so that I can add jar files in web.jsf12/external folder?
1. commons-beanutils.jar
2. commons-collections.jar
3. commons-digester.jar
4. jsf-impl.jar